assets/assets/utils/minio_utils.py

```diff
@@ -262,24 +218,17 @@ def make_thumbnail_pdf(file: bytes) -> Union[bool, bytes]:
     return byte_im
 
 
-def make_chemical_thumbnail(file: bytes, fmt: str) -> Union[bool, bytes]:
-    """
-    Handles thumbnail creation for chemicals files 
-    with extension .mol , .sdf
-    :param file_format : sdf, mol
-    """
-    buf = BytesIO()
-    img = None    
-    img = read_chemical_page(file, page_number=0, file_format=fmt)
-
-    if img is None:
-        return False   
-    size = thumb_size(img)
-    img.thumbnail(size)
-    img.save(buf, format="png")
-    img.close()
-    byte_im = buf.getvalue()
-    return byte_im
+def make_thumbnail_chem(file: bytes) -> Union[bool, bytes]:
+    logger_.info("Generating thumbnail from molecules")
```

Comment:
Maybe worth making it `debug` or provide additional info, such as file name or something else?

Reply:
Yep, agree